Add alternative case for hillsideview46

daniel2887 opened the pull request:
Following up on our Reddit chat :)

Everything is isolated to a single directory. I left it up to you if/how you want to call out this alternative case from elsewhere in your docs.

daniel2887 committed:
Add alternative case for hillsideview46

dalewking commented:
Would be nice if the alternative case was compatible with the tenting stand

dalewking commented:
@daniel2887 Any way you could do a version for the 3x5 configuration?

dalewking commented:
@daniel2887 Can you also specify the correct part number for the trackpad?

daniel2887 committed:
Add the Fusion 360 project for the hillsideview46 alternative case

dalewking commented:
I have a suggestion which I think I will try. I do not like the way the power switch is visible on the left board and is under the cirque and hard to operate on the right board. Instead of mounting the power switch on the top of the board mount it on the bottom of the board, close up the hole on the top of the case and instead add a slot to the side of the case to access the switch

daniel2887 committed:
Add alternative case for bottom mounted components

Some users prefer to solder the power switch and FFC connector
to the bottom of the PCB. This commits adds compatible versions of the
alternative case.

daniel2887 commented:
@dalewking following up on our Reddit chat: See the new directory, `PowerAndFFCOnBottom` and let me know if this works for you. Note that I didn't print this out, but it _should work_ (famous last words).

Regarding your other comments:

> Would be nice if the alternative case was compatible with the tenting stand

I have no plans to follow up on this, but the complete Fusion 360 project is pushed here as well if anyone wants a go at this.

FWIW, for my tenting solution I wanted something more integrated. I use these [tenting feet](https://typeractive.xyz/products/tenting-feet), which work very well.

> Any way you could do a version for the 3x5 configuration?

I haven't had time to follow up on this, but I can give it a try (but, again, I won't be able to test this) if you still need this. Let me know.

> Can you also specify the correct part number for the trackpad?

This should probably be part of the parent repo docs, not this alternative case PR. But in case you need this info for your build, see [this link](https://www.mouser.com/ProductDetail/355-TM0400402024-303), from this [Reddit thread](https://www.reddit.com/r/ErgoMechKeyboards/comments/1e2qv1w/comment/ld3np7s/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button).

dalewking commented:
The placement of the slot for the power switch looks good, might be nice to chamfer the edges of the slot.

The FFC change on the bottom pieces looks good, but is way bigger than it needed to be. All you needed was a a couple of millimeters but it will work.

However you moved the slot for the ribbon on the top right piece and that definitely is not correct. The connector on the top right should still be on top. My issue was that I had an unused FFC connector on the bottom of the left board that interfered with the bottom case on that side.

It may be December before I will be printing the top covers due to budgeting for it (I don't have a 3d printer and print at the library and I need to buy some other parts first).

> FWIW, for my tenting solution I wanted something more integrated. I use these [tenting feet](https://typeractive.xyz/products/tenting-feet), which work very well

Was planning to go this way too

> >Any way you could do a version for the 3x5 configuration?
>
> I haven't had time to follow up on this, but I can give it a try (but, again, I won't be able to test this) if you still need this. Let me know.

I've decided to use the 3x6 so I don't need this anymore

> This should probably be part of the parent repo docs, not this alternative case PR. But in case you need this info for your build, see [this link](https://www.mouser.com/ProductDetail/355-TM0400402024-303), from this [Reddit thread](https://www.reddit.com/r/ErgoMechKeyboards/comments/1e2qv1w/comment/ld3np7s/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button).

Yeah, I figured it out already by measuring the size of the parts
